visivo/logging/logger.py

```diff
@@ -37,23 +37,16 @@ def info(self, message: str):
         if self.echo:
             self.echo(message)
         elif self.spinner:
-            self.spinner.write(message)
+            self.spinner.info(text=message)
 
     def success(self, message: str):
         if self.echo:
             self.echo(message)
         elif self.spinner:
-            self.spinner.succeed(message)
+            self.spinner.succeed(text=message)
 
     def error(self, message: str):
         if self.echo:
             self.echo(message)
         elif self.spinner:
             self.spinner.stop_and_persist(text=message)
```

Comment:
I think there's an `spinner.error` halo object that we can use rather than stop and persist. I think that if we use the error object we'll get some nice formatting benefits.

Reply:
Done.